README.md

```diff
@@ -118,6 +118,22 @@ To stop use:
 make inventory-down
 ```
 
+#### Kessel Inventory + Kessel Relations using built binaries
+
+Inventory and Relations can also be run locally using built binaries, but the default config for Inventory will conflict with Relations.
+
+To run Relations locally, see the [Running Locally](https://github.com/project-kessel/inventory-api/tree/main?tab=readme-ov-file#running-locally) section of the Relations README
```

Comment:
This description mentions seeing the relations readme, but points to the inventory-api readme.

Reply:
🤦🏻